scripts/scil_compute_metrics_stats_in_ROI.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Compute the statistics (mean, stddev) of scalar maps, which can represent
+diffusion metrics, in a ROI.
+
+The mask can either be a binary mask, or a weighting mask. If a weighting mask
+should either contain floats between 0 and 1, or should be normalized with
```

Comment:
If it is a weighting mask, it should...

Or at least add a comma? If a weighting mask, should.... But that's computerish, not english.